src/stories/MessageModal.stories.tsx

```diff
@@ -43,7 +43,7 @@ export const Default: Story = {
 
 export const WithHeader: Story = {
   render: (args) => {
-    const [open, setOpen] = useState(true);
+    const [open, setOpen] = useState(false);
```

Comment:
When the Story Docs were put together, there was chaos with various modals opening at the same time.
Except for one, the initial view is closed.